interop-testing/src/main/java/io/grpc/testing/integration/AbstractInteropTest.java

```diff
@@ -1472,6 +1476,7 @@ public void customMetadata() throws Exception {
   @Test(timeout = 10000)
   public void censusContextsPropagated() {
     Assume.assumeTrue("Skip the test because server is not in the same process.", server != null);
+    Assume.assumeTrue(metricsExpected());
```

Comment:
Same reason as above. This should be `assumeTrue(serverBuilderIsAbstractServerImplBuilder)`.